v2_papers/2009/kind_histogram.json

```diff
@@ -0,0 +1,179 @@
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured"
+      },
+      {
+        "name": "data center",
+        "types": [],
+        "generation_year": 2007,
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "covered_period": "months"
+      },
+      {
+        "name": "DARPA IDeval",
+        "availability": "public",
+        "format": "packet",
+        "types": [],
+        "generation": "altered_captured",
+        "generation_year": 1999,
+        "covered_period": "weeks",
+        "details": [
+          "raw"
+        ],
+        "subsets": []
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": true,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [],
+    "normalization_type": "no",
+    "transformations": [
+      "_histograms",
+      "_metric_space",
+      "entropy"
+    ],
+    "final_data_format": "_histograms",
+    "feature_selections": [
+      {
+        "name": "constant-removal",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      },
+      {
+        "name": "pca",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      }
+    ],
+    "flows": [
+      {
+        "selection": "study_based",
+        "role": "main",
+        "main_goal": "anomaly_detection",
+        "active_timeout": "missing",
+        "idle_timeout": "missing",
+        "features": [
+          "sourceIPv4Address",
+          "destinationIPv4Address",
+          "sourceTransportPort",
+          "destinationTransportPort",
+          "protocolIdentifier",
+          "octetTotalCount",
+          "packetTotalCount",
+          "flowDurationMilliseconds"
+        ],
+        "key_features": []
+      }
+    ]
+  },
+  "analysis_method": {
+    "supervised_learning": true,
+    "unsupervised_learning": true,
+    "semisupervised_learning": false,
+    "anomaly_detection": true,
+    "tools": [],
+    "algorithms": [
+      {
+        "name": "hierarchical cluster analysis",
+        "subname": "hca",
+        "learning": "unsupervised",
+        "role": "intermediate",
+        "type": "clustering",
+        "metric/decision_criteria": "euclidean",
+        "source": "referenced",
+        "parameters_provided": false
+      },
+      {
+        "name": "clustering",
+        "subname": "K-means",
+        "learning": "unsupervised",
+        "role": "intermediate",
+        "type": "clustering",
+        "metric/decision_criteria": "euclidean",
+        "source": "referenced",
+        "parameters_provided": false
+      }
+    ]
+  },
+  "evaluation": {
+    "algorithm_comparison": false,
+    "internal_validation": true,
```

Comment:
You say they use internal validation, but present no internal method. Could you add the method you mean? (or set this to false, if it was a mistake)

Reply:
They used clustering to label the histograms but I am not sure if HCA is correct (IV.C)

Reply:
Not sure I understand what you mean. This section is for the metrics used to evaluate the methods, not for the methods themselves. Meaning, if you use for example K-means and only evaluate its accuracy, there is no internal validation. Internal validation would be metrics that don't use labels (e.g., silhouette).